### PR TITLE
Add are you alive example

### PR DESCRIPTION
**Pull request by J0**
This Pull Request aims to add an example of how to set up an are-you-alive deployment for Vitess. This is open sourced from PlanetScale's internal version of Are-You-Alive.

Signed-off-by: Lee Yi Jie Joel <lee.yi.jie.joel@gmail.com>

### CODE REVIEW

**Comment by deepthi** [comment]
@sverch @aquarapid can you review this?

**Comment by aquarapid** [comment]
LGTM

File: examples/are-you-alive/Makefile [hunk 1]
```diff
@@ -0,0 +1,15 @@
+# We are using the "dev" project in our registry for this
+NAME   := "us.gcr.io/planetscale-dev/are-you-alive"
```
Comment (sverch) [inline examples/are-you-alive/Makefile]: I think this should be `us.gcr.io/planetscale-vitess/are-you-alive`.  Does that sound good to you @dctrwatson?

Comment (dctrwatson) [reply examples/are-you-alive/Makefile]: If this is meant for public consumption, yes :)

Comment (sverch) [reply examples/are-you-alive/Makefile]: Ok, updated, but now I need to fix the build.

**Comment by sverch** [comment]
@deepthi Is this good to merge?  One test timed out but it looks unrelated to this change and isn't listed as required.

File: examples/are-you-alive/deploy/README.md [hunk 2]
```diff
@@ -0,0 +1,5 @@
+# Kubernetes Deployment
+
+Configuration for deploying `are-you-alive` on Kubernetes.
+
+Also deploys Prometheus and Alertmanager.
```
Comment (deepthi) [inline examples/are-you-alive/deploy/README.md]: are we really deploying AlertManager? I don't see it.

File: examples/are-you-alive/README.md [hunk 3]
```diff
@@ -0,0 +1,82 @@
+# Are You Alive?
+
+What does it mean to be alive?
+
+Well we don't know what it means for you, but we know what it means for our
+Cloud Database!
+
```
[the remark this answers is not in the document]
Comment (sverch) [reply examples/are-you-alive/README.md]: Changing from "Cloud Database" to "Vitess Cluster".

File: examples/are-you-alive/README.md [hunk 4]
```diff
@@ -0,0 +1,82 @@
...
+After you run docker compose, navigate to `http://localhost:9090` to see
+Prometheus and `http://localhost:8080/metrics` to see the raw metrics being
+exported.
+
+## Push to Registry
+
+```
+make build
+make push
+```
```
[the remark this answers is not in the document]
Comment (sverch) [reply examples/are-you-alive/README.md]: I'll add a note about access here, but I think documenting the commands makes sense.

File: examples/are-you-alive/deploy/README.md (hunk 2 above)
Comment (sverch) [reply examples/are-you-alive/deploy/README.md]: Ah, thanks for calling this one out.  I'll just delete this directory.

**Comment by sverch** [comment]
Thanks @deepthi !  I think I addressed your comments.  Let me know if there's anything else I should do before this is good to merge.

**Pull request merged**